app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
- [ ] Kindly remove the `foreign_key: 'author_id'` and `foreign_key: 'post_id'`. Remember that the `User` has_many `Comments`, therefore the `foreign_key` will be on the `Comments` Table. The same applies to the relationship between the `Post` and the `Comment` 👍